packages/debugger/src/debugger.ts

```diff
@@ -100,12 +100,14 @@ export namespace Debugger {
    * The command IDs used by the debugger plugin.
    */
   export namespace CommandIDs {
-    export const debugContinue = 'debugger:continue';
+    export const pauseOrContinue = 'debugger:pause-continue';
 
     export const terminate = 'debugger:terminate';
 
     export const next = 'debugger:next';
 
+    export const pause = 'debugger:pause';
+
```

Comment:
Could you restore the `debugger:continue` command?
And I think the `pause` removal is missing from the latest changes.

This will ease backport as the command ids are part of the public API.

Reply:
Sure, thank for review @fcollonval . I restored the command name in https://github.com/jupyterlab/jupyterlab/pull/13433/commits/093de33867084d6659be9a5f9233fcde93118a7b, but kept the pause feature in this command.